packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_ETH.sol

```diff
@@ -29,4 +30,47 @@ contract OVM_ETH is L2StandardERC20 {
             "ETH"
         )
     {}
+
+
+    /******************************
+     * Custom WETH9 Functionality *
+     ******************************/
+
+    /**
+     * Implements the WETH9 deposit() function as a no-op.
+     * WARNING: this function does NOT have to do with cross-chain asset bridging. The
+     * relevant deposit and withdraw functions for that use case can be found at L2StandardBridge.sol.
+     * This function allows developers to treat OVM_ETH as WETH without any modifications to their code.
+     */
+    function deposit()
+        public
```

Comment:
```suggestion
        external
```

Probably doesn't make a difference, but `external` is more restrictive and should be prefferred whenver you don't also need to call a function with a JUMP.

Reply:
Couldn't make `deposit` external, since I added a fallback which calls it